### PR TITLE
TDR-2752 file checks results success content changes

### PR DESCRIPTION
**Pull request by thanhz**

### CODE REVIEW

File: app/views/standard/fileChecksResults.scala.html [hunk 1]
```diff
@@ -8,23 +8,32 @@
...
+                        Your folder '@{consignmentInfo.parentFolder}' containing @{consignmentInfo.numberOfFiles} item@if(consignmentInfo.numberOfFiles > 1){s} has been successfully uploaded.
+                    </h3>
+                    <p class="govuk-body">You can leave and return to this transfer at any time from the <a class="govuk-notification-banner__link" href="@routes.ViewTransfersController.viewConsignments()">View transfers</a> page.</p>
+                </div>
+            </div>
+            <div class="govuk-warning-text">
+                <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+                <strong class="govuk-warning-text__text">
+                    <span class="govuk-warning-text__assistive">Warning</span>
+                    Now that your records have been uploaded you can proceed with the transfer. In the next step you will be given the opportunity to add metadata to your records before transferring them.</strong>
```
Comment (MancunianSam) [inline app/views/standard/fileChecksResults.scala.html:28]: The text about having "the opportunity to add metadata" is only applicable if `blockClosureMetadata` or `blockDescriptiveMetadata` is false. Might be worth double checking with @garethfoote but it looks like this sentence should be inside an `@if(blockClosureMetadata || blockDescriptiveMetadata)`

**Review by MancunianSam — COMMENTED** [review]
Looks good, cheers. Just one question about the text if the metadata is turned off.

File: app/views/standard/fileChecksResults.scala.html (hunk 1 above)
Comment (thanhz) [reply app/views/standard/fileChecksResults.scala.html]: Spoke with Gareth. The text "..the opportunity to add metadata" will not be displayed if metadata is turned off. I've made the relevant changes.

**Review by MancunianSam — APPROVED** [review]
:+1:

**Pull request merged**